src/PIL/ImageQt.py

```diff
@@ -213,4 +213,7 @@ def toqimage(im: Image.Image | str | QByteArray) -> ImageQt:
 
 def toqpixmap(im: Image.Image | str | QByteArray) -> QPixmap:
     qimage = toqimage(im)
-    return getattr(QPixmap, "fromImage")(qimage)
+    pixmap = getattr(QPixmap, "fromImage")(qimage)
+    if qt_version == "6":
```

Comment:
This "6" means PyQt, and "side6" would mean PySide6. 

Do we know if this also affects PySide6?

Reply:
If I run the test suite [with PySide6](https://github.com/radarhere/Pillow/commit/ded6ccaad73731e56053fd6314207799a91a1d0b) without this fix, [it passes.](https://github.com/radarhere/Pillow/actions/runs/11563904843)

Furthermore, [`detach()`](https://www.riverbankcomputing.com/static/Docs/PyQt6/api/qtgui/qpixmap.html#detach) doesn't exist in PySide6 - https://doc.qt.io/qtforpython-6/PySide6/QtGui/QPixmap.html / https://github.com/python-pillow/Pillow/actions/runs/11554805812/job/32158808074#step:6:5698. It is possible they don't offer it precisely because it isn't needed?

Reply:
Thanks for testing, let's merge. We'll have to see what happens when/if PyQt7 comes out in the future.